src/pytest_molecule/__init__.py

```diff
@@ -173,6 +173,13 @@ def __init__(self, name, parent):
             self.molecule_driver = data.get("driver", {}).get("name", "no_driver")
             self.add_marker(self.molecule_driver)
 
+            # check for tags and add them as markers
+            for tag in data.get("tags", []):
+                try:
+                    self.add_marker(tag)
+                except pytest.PytestUnknownMarkWarning:
+                    pass
```

Comment:
I am not sure how I feel about silently ignoring invalid marks. What is the use case for ignoring invalid ones? I would expect pytest-molecule users to take care of their testing setup and thus should be notified when something went awry.

Reply:
I used tags on purpose in order to match ansible as I expect to make use of that for other purposes in the future.

We ignore unknown tags because pylint complains about them. In fact my impression is that they may even become failures in the future. Those that want to use custom marks they need to document them in pytest config file, with description.

Reply:
We shouldn't ignore such markers. Pytest has a built-in mechanism for turning them into errors `--strict-markers` and it must remain supported for these too.

Reply:
If we are to not ignore unknown ones, we should add something like `pytest-markers: []` instead of generic `tags: []`, which we could reuse for other features.

My personal take on this is to go for multi-purpose tags/labels like approach, similar to ansible one, where special values have meaning but were you can still use arbitrary values without having to worry that it will break code.

Still, if most are against this approach, I will not try to force my opinion on them.

Probably you observed that the goal was to allow use of `xfail` and `xskip`.